src/mono/mono/mini/debugger-protocol.h

```diff
@@ -0,0 +1,374 @@
+#ifndef __MONO_DEBUGGER_PROTOCOL_H__
+#define __MONO_DEBUGGER_PROTOCOL_H__
+
+#include "glib.h"
+
+#define HEADER_LEN 11
+#define REPLY_PACKET 0x80
+
+
```

Comment:
Extra empty lines here and many other places.

Reply:
Fixed.